Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -326,8 +326,26 @@ namespace EMotionFX
         {
             return;
         }
+        if (!m_editor->HasDisplayedNodes())
```

Comment:
bool ColliderWidget::HasDisplayedNodes() const
You should use this function instead.